packages/react-scripts/backpack-addons/splitChunks.js

```diff
@@ -4,23 +4,34 @@ const paths = require('../config/paths');
 const appPackageJson = require(paths.appPackageJson);
 const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
 
-module.exports = (isEnvDevelopment) => {
+module.exports = isEnvDevelopment => {
```

Comment:
Is this formation changed by `husky` hooks when executing `git` commands?

Reply:
This is due to formatting the document, and I will restore it.